extensions/rvdatabinding/src/main/java/com/google/android/agera/rvdatabinding/DataBindingRepositoryPresenterCompilerStates.java

```diff
@@ -65,6 +68,21 @@
     DBRPMain<T> onRecycle(@RecycleConfig int recycleConfig);
 
     /**
+     * Specifies a data binding @{code itemId} from the previously given {@code layout} to bind a
+     * single item in the {@link Repository}.
+     */
+    @NonNull
+    RPCollectionCompile<T> collectionId(int collectionId);
+
+    /**
+     * Specifies a {@link Function} to return a data binding @{code itemId} from the previously
+     * given {@code layout} to bind a single item in the {@link Repository}.
+     */
+    @NonNull
+    <TCol> RPTypedCollectionCompile<T, TCol> collectionIdForCollection(
```

Comment:
let's remove this until there's usage. I'm also tempted to remove the itemIdForItem() option too, but it's already out.